erikbenedict/README-generator
# Feature/generate markdown

## TEST

- make sure all packages are installed
- open index.js in integrated terminal
- run node index.js
- complete all questions propmted
- see that exampleREADME.md has been updated and reflects user input in a syntactically correct markdown file
